Sources/SmileID/Classes/DocumentVerification/Model/DocumentCaptureModel.swift

```diff
@@ -350,10 +390,18 @@ class DocumentCaptureViewModel: ObservableObject, JobSubmittable, ConfirmationDi
                 let rect = CGRect(origin: transparentRectOrigin, size: self.guideSize)
                 if self.isWithinBoundsAndSize(cgrect1: rect, cgrect2: transformedQuad.cgRect) && self.textDetected {
                     self.borderColor = SmileID.theme.success.uiColor()
-                    self.showCaptureButton = true
+                    if self.autoCaptureTimer == nil {
+                        self.showCaptureButton = false
+                        self.autoCaptureTimer = Timer.scheduledTimer(timeInterval: 2,
```

Comment:
Can we lift up the times into constants? both here and for the manual capture timer?